.github/ISSUE_TEMPLATE/other.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Other
+about: For issues that don't fit into any of the above categories.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+SPARKLERS BEWARE! THIS SPARKLETOWN/SPARKLE REPO IS PUBLIC TO THE ENTIRE INTERNET. ARE YOU SURE YOU WANT TO BE OPENING AN ISSUE HERE AND NOT IN SPARKLETOWN/INTERNAL-SPARKLE-ISSUES INSTEAD?
+-->
```

Comment:
[note] In combination with the change in `.github/ISSUE_TEMPLATE/config.yml` to prevent creating blank issues, this generic 'other' template is basically blank, except still includes the warning about creating issues on the public repo.